app/src/main/res/values/strings.xml

```diff
@@ -211,4 +211,5 @@ be temporary, and you can try again later.</li>
     %1$s will be the title of the recommended video
     %2$s will be the source of the video (e.g. YouTube). -->
     <string name="pocket_video_feed_a11y_feed_item_focused">%1$s on %2$s</string>
+    <string name="remove_tile_toast_title">Press and hold SELECT to unpin from homescreen</string>
```

Comment:
nit: I prelanded landed this with our string export – can you use that string instead?